signbank/dictionary/templates/dictionary/gloss_detail.html

```diff
@@ -191,9 +191,9 @@ <h4 class="edit edit_text" id="video_title{{glossvideo.pk}}">{{glossvideo.title}
                                 <video id="glossvideo-{{glossvideo.pk}}" class="embed-responsive-item" width="450" preload="metadata" controls muted
                                        {% if glossvideo.posterfile %} poster="{{glossvideo.posterfile.url|urlencode}}"{% endif %}>
                                     {% if glossvideo.get_extension == '.mp4' %}
-                                    <source src="{{glossvideo.videofile.url|urlencode}}" type="video/mp4">
+                                    <source src="{{glossvideo.videofile.url}}" type="video/mp4">
```

Comment:
I'm unsure why the URL was being URL-encoded - since it's calling the file storage provider's `url` method, it should _already_ be URL encoded.